js/main.js

```diff
@@ -43,11 +44,11 @@ $submitForm.addEventListener('submit', function (event) {
         $liElements[i].replaceWith(renderEntry(formData));
       }
     }
-
     $entryTitle.textContent = 'New Entry';
-    viewSwap('entries');
     data.editing = null;
   }
+  viewSwap('entries');
+  toggleNoEntries();
```

Comment:
You should call `$submitForm.reset()` here instead. This way we always reset the form after creating a new entry or editing an entry